src/Components/Samples/BlazorServerApp/Pages/_Layout.cshtml

```diff
@@ -0,0 +1,28 @@
+﻿@namespace BlazorServerApp.Pages
+@using Microsoft.AspNetCore.Components.Sections
```

Comment:
```suggestion
```

... if we move the head components into `.Web`

Reply:
It doesn't seem that `_Imports.razor` applies to .cshtml files, is that correct? In which case I would just have to change this to

```suggestion
@using Microsoft.AspNetCore.Components.Web
```

Reply:
Good point!